ontology/ontoneo.md

```diff
@@ -2,32 +2,43 @@
 layout: ontology_detail
 id: ontoneo
 title: Obstetric and Neonatal Ontology
-browsers:
-- title: BioPortal Browser
-  label: BioPortal
-  url: https://bioportal.bioontology.org/ontologies/ONTONEO
-build:
-  source_url: http://purl.obolibrary.org/obo/ontoneo/ontoneo.owl
 contact:
   email: fernanda.farinelli@gmail.com
   github: FernandaFarinelli
   label: Fernanda Farinelli
   orcid: 0000-0003-2338-8872
 description: The Obstetric and Neonatal Ontology is a structured controlled vocabulary to provide a representation of the data from electronic health records (EHRs) involved in the care of the pregnant woman, and of her baby.
-domain: health
-homepage: ontoneo.com
 license:
-  label: CC BY 3.0
-  url: http://creativecommons.org/licenses/by/3.0/
+  label: CC BY 4.0
+  url: https://creativecommons.org/licenses/by/4.0
+domain: biomedical, health, obstetric and neonatal domain, gynecological domain, women's 
```

Comment:
`domain` should be one of the following:
- 'upper'
- 'anatomy and development'
- 'health'
- 'phenotype'
- 'investigations'
- 'environment'
- 'chemistry and biochemistry'
- 'microbiology'
- 'agriculture'
- 'diet, metabolomics, and nutrition'
- 'biological systems'
- 'information technology'
- 'organisms'
- 'information'
- 'simulation'

In the case of Ontoneo, the domain should be 'health'

Reply:
```suggestion
domain: health
```